pages/signal-monitor.js

```diff
@@ -45,6 +45,7 @@ export default function SignalMonitor() {
         title="Signal monitor"
         getMapIcon={getMapIcon}
         featurePk="signal_id"
+        noFeaturesMessage="Signal system operating normally."
```

Comment:
Does the absence of this new prop need to be handled in the traffic cameras and signal eval maps? It doesn't seem like it to me, but I wanted to check. Maybe this ties into a future request error state feature too.

Reply:
I was thinking no since there should never be zero features for the signal eval map and the traffic cameras, unless there was a data retrieval failure and it would break before then.

Reply:
I think it would be good to add a `noFeaturesMessage` to the other dashboards, just to be safe. and perhaps to be extra safe not render the alert banner at all if there is not a `noFeaturesMessage`, or at least make sure it doesn't break if no message is provided?

Reply:
also, channeling my inner Amenity, i think you should remove the period (`.`) from the message